3Dmol/glmodel.js

```diff
@@ -2462,11 +2462,8 @@ $3Dmol.GLModel = (function() {
                     for (var i = 0; i < numFrames; i++) {
                         $3Dmol.getbin("http://"+url+"/traj/frame/"+i+"/"+path, function (buffer) {
                             self.setCoordinates(buffer, "unknown");
-                            viewer.setStyle({},{sphere:{}});
-                            viewer.zoomTo();
-                            viewer.animate({loop:"forward",reps:1});
-                            viewer.render();
-                        },'POST');
+                            self.setFrame(i) 
```

Comment:
setFrame should not be called from within setCoordinatesFromURL

The user called setCoordinatesFromURL _or_ setCoordinates and then they can call setFrame or animate or any other function that changes the frame and it should work as expected.  If the coordinates were initialized from a URL, the code will dynamically fetch the coordinates on demand.  Initially this can be implemented as a synchronous request, but should eventually be implemented with generators (https://eladnava.com/write-synchronous-node-js-code-with-es6-generators/).